README.rst

```diff
@@ -20,4 +20,98 @@ Python implementation of PVL (Parameter Value Language)
 Features
 --------
 
-* TODO
\ No newline at end of file
+* TODO
```

Comment:
Yeah, I was planing on doing some thinking about explaining what pvl is, where it came from and the approach we're taking in accepting the various "dialects".